packages/fxa-auth-server/scripts/recorded-future/check-and-reset.ts

```diff
@@ -266,6 +270,52 @@ async function getAccountsByLogin(logins: SearchResultIdentity[]) {
   return { accounts, has2faCount };
 }
 
+async function resetAccounts(
+  accountsToReset: ResetableAccount[],
+  resetWithEmails: boolean
+) {
+  const authDb = await getAuthDb();
+  const bounces = bouncesFn(config, authDb);
+  const senders = await sendersFn(log, config, bounces, statsd);
+  // the dynamically named `send\w+Email` functions are not in the type of senders.email
+  const mailer: any = senders.email;
+
+  for (const acct of accountsToReset) {
+    try {
+      await authDb.resetAccount(
+        { uid: acct.uid },
+        {
+          authSalt: butil.ONES.toString('hex'),
+          verifyHash: butil.ONES.toString('hex'),
+          wrapWrapKb: crypto.randomBytes(32).toString('hex'),
+          verifyHashVersion2: butil.ONES.toString('hex'),
+          wrapWrapKbVersion2: crypto.randomBytes(32).toString('hex'),
+          verifierVersion: 1,
+        }
+      );
+      await oauthDb.removeTokensAndCodes(acct.uid);
+      await mailer.sendPasswordChangeRequiredEmail(acct.emails, acct);
+
+      if (!resetWithEmails) {
+        statsd.increment(
```

Comment:
Wouldn't you still want to emit a metric even resetWithEmails is true?

Reply:
I was thinking the email param is mainly for testing so we needn't emit metrics for it.  But it wouldn't hurt to have.  I'll add it.